package.json

```diff
@@ -9,7 +9,8 @@
     "dev": "NODE_ENV=development npm run start",
     "dev:search": "APP_NAME=search npm run dev",
     "dev:disco": "APP_NAME=disco npm run dev",
-    "test": "npm run version-check && npm run lint",
+    "test": "npm run version-check && npm run karma",
```

Comment:
I was thinking that but running `npm test ` for the tests seemed reasonable and eslint isn't super fast. We should probably have a "test all the things" script though.